frontend/src/lib/candidate/components/PasswordField/PasswordField.svelte

```diff
@@ -4,6 +4,7 @@
   export let password = '';
   export let autocomplete = '';
   export let id = 'password';
+  $: fieldId = id;
```

Comment:
What is the purpose of this? I'd assume Svelte automatically re-renders when the props change. If there is a good reason, an inline comment should be added